examples/jsm/webxr/XRControllerModelFactory.js

```diff
@@ -14,7 +14,7 @@ import {
 	MotionController
 } from '../libs/motion-controllers.module.js';
 
-const DEFAULT_PROFILES_PATH = 'https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@1.0/dist/profiles';
+const DEFAULT_PROFILES_PATH = 'https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@^1.0/dist/profiles';
```

Comment:
@toji Are you okay with this change?

Reply:
Yeah! This was my intent from the beginning, but I'm a bit confused about the syntax. According to the sample URLs on the jsdelivr.com main page I thought simply leaving off the patch revision would select the latest in that range?

Reply:
I guess it is the same?

<img width="364" alt="Screen Shot 2020-10-27 at 5 22 50 PM" src="https://user-images.githubusercontent.com/97088/97338120-7c1f1300-18c4-11eb-8ee4-aea3c6c1cd2b.png">

Reply:
Confusingly, it doesn't appear to be the same behavior. I don't understand why either, given their examples.

https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@1/dist/profiles/profilesList.json
https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@1.0/dist/profiles/profilesList.json
https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@1.x/dist/profiles/profilesList.json

None of these include the v3 profile

If you go to https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets/, it only shows the latest available version as 1.0.2.

It looks like this might be because the latest version of the package on npm is 1.0.2? The registry shows it was last updated 7 months ago https://www.npmjs.com/package/@webxr-input-profiles/assets. But jsDelivr definitely has 1.0.4 https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@1.0.4/dist/profiles/profilesList.json

How was jsDelivr being updated if the package wasn't published to npm?

Reply:
The package was un-published from NPM temporarily due to a miscommunication about when it was ready to publish. I'll push it out again as 1.0.5 when that's resolved.

Reply:
To be clear, this is still a good change whether or not a new package is currently published. The intended behavior is that it picks up new assets automatically as they get published, so landing this now will better prepare the library for future pushes.

Reply:
I think the original syntax will work and jsDelivr is working as intended, there was just a timing bug with the package being unpublished. The URL in this PR, the one that worked for me last night -- https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@^1.0/dist/profiles/profilesList.json -- now no longer returns a list with the Touch v3 profile; I believe it's using 1.0.2 like the rest of the URLs I listed.

Reply:
Although, playing with their examples some more, we will need to change the URL to either `1` `1.x` or `^1.0` to automatically pick up minor versions. As it stands it'll only pick up patch versions.

Is there a preference for which format is used?